cl/search/tasks.py

```diff
@@ -1112,6 +1117,18 @@ def index_parent_or_child_docs(
                 )
             elif document_type == "child":
                 child_instances = Opinion.objects.filter(pk__in=instance_ids)
+                # Get unique parent_ids for Opinions
+                parent_ids = list(
+                    set(
+                        Opinion.objects.filter(
+                            pk__in=instance_ids
+                        ).values_list("cluster_id", flat=True)
+                    )
```

Comment:
is there a reason to use the set constructor instead of the [distinct method](https://docs.djangoproject.com/en/5.0/ref/models/querysets/#distinct) ?

Reply:
Yeah, I changed updated this too!